roles/mailserver/files/nginx/www-poweradmin.conf

```diff
@@ -0,0 +1,37 @@
+server {
+  listen 8000;
+  server_name poweradmin.{{ domain_name }};
+
+  include /conf.d/robots;
+
+  location / {
+    proxy_pass http://poweradmin;
+    include /etc/nginx/conf/proxy_params;
+  }
+}
+
```

Comment:
Does the `pdns` UI even have an SSL (443) endpoint? If not (which I don't think it does, looking at the Dockerfile for it), you can just remove the second `server` stanza below.